pkg/frontend/adminactions/kubeactions.go

```diff
@@ -43,7 +43,7 @@ type kubeActions struct {
 	log *logrus.Entry
 	oc  *api.OpenShiftCluster
 
-	gvrResolver dynamichelper.GVRResolver
+	gvrResolver dynamichelper.GVRInterface
```

Comment:
Changed so that we use the interface rather than the implementation

Reply:
I am not sure I see what that does, GVRResolver was already an interface, so we were using the interface rather than the implementation